app/models/changelog.php

```diff
@@ -122,3 +122,20 @@
 
     return $text;
 };
+
+$latest_undocumented_release = function() use ($releases, $github_link, $release_title) {
+  $text = '';
+  $version_number = str_replace("\n", '', str_replace('v', '', file_get_contents(CACHE_PATH . 'tag.txt')));
+  end($releases);
+  $last_release = key($releases);
+
+  if ($version_number != $last_release) {
+    $github_relnotes = "https://github.com/mozfr/transvision/releases/tag/v{$version_number}";
+    $releases[$version_number] = 'Unknown';
+    $text .= $release_title($version_number, $releases);
+    $text .= "<p>This release is undocumented, but you can know more in the <a href=\"{$github_relnotes}\">GitHub release notes</a> and with the list of commits below since the last documented release.</p>";
```

Comment:
Maybe: There is no entry in the changelog for this release, but you can learn more about it in the GitHub release notes and look at the list of commits from the latest documented release.